codemon/CodemonMeta.py

```diff
@@ -1,36 +1,48 @@
+import os
 from clint.textui import colored
 from datetime import datetime
 
+def copy_template(filepath):
+  with open(filepath) as fp:
+    line = fp.readline()
+    stringArr = []
+    while line:
+      stringArr.append(line.strip())
+      line = fp.readline()
+    return "\n".join(stringArr)
+
+
+def template_exists(directory):
+  for filename in os.listdir(directory):
+    if filename == "template.cpp":
+      return True
+  return False
+
 def template_cpp():
   # get the current time
-  now = datetime.now()
+  # now = datetime.now()
   # the template for initial code
-  template = """/*
-  author: @ankingcodes
-  created: %s
-*/
-#include<bits/stdc++.h>
-#include<algorithm>
-using namespace std;
-
-#define ll long long
-#define MOD 1000000000
-
-int main(){
-  ios_base::sync_with_stdio(false);
-  cin.tie(NULL);
-  ll t;
-  return 0;
-}
-""" % (now)
+  template = """
+    #include<bits/stdc++.h>
+    using namespace std;
+
+    int main() {
+  
+      return 0;
+  
+    }
+  """

```

Comment:
Okay, So the default is what it is previously was. But the template.CPP could be anything as per the user.
Well, I had tested in my local machine, I think I was not able to understand the issues related to alignment.

Reply:
@yashshah14093 - This template is just an alternative for users who don't have a template at `~/.codemon`. Note that the name should not be fixed to `template.cpp`, it could be any name but have the string template in it. eg `codejam_template.cpp`.
Regarding the alignment, you'd note that the spaces given before each line of the strings would create a file with improper identation. Also, you need not alter the template.